platform/linux/linux.gradle

```diff
@@ -7,11 +7,16 @@ tasks.register('exportLinux64', Copy.class) {
     dependsOn jar
     dependsOn exportPlugins
 
+    def icon = "icon.png"

```

Comment:
Please make solution that always uses icon.png and renames the file instead as it is cleaner.

Renaming the file is not a problem in Gradle

Reply:
I don't get what you mean...

Reply:
It's done, I did what you said...